plugins/forc-index/src/commands/deploy.rs

```diff
@@ -65,6 +65,10 @@ pub struct Command {
     /// Do not build before deploying.
     #[clap(long, help = "Do not build before deploying.")]
     pub skip_build: bool,
+
+    /// Stop the previous running indexer.
```

Comment:
I mentioned this in the last review, but while this blurb makes sense to us, I don't think it's too informative for someone who's unfamiliar with the context of deploying indexers.

```suggestion
    /// Stop the currently running version of this indexer, if one exists. 
```
Or some such

Reply:
Ok that's in!